composer.json

```diff
@@ -31,7 +31,9 @@
         "php": "^7.2 || ^8.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0 || ~9.4.4"
+        "phpunit/phpunit": "^8.0 || ~9.4.4",
+        "friendsofphp/php-cs-fixer": "^3.13",
+        "phpstan/phpstan": "^1.9"
```

Comment:
I'll be okay.
Complete CI comes in the next PR.

Reply:
No need for these changes, please remove.